src/parser/view_transform.c

```diff
@@ -12029,3 +12976,79 @@ mq_copy_view_error_msgs (PARSER_CONTEXT * parser, PARSER_CONTEXT * query_cache)
 
   pt_record_error (parser, stmt_no, line_no, col_no, msg, NULL);
 }
+
+/*
+ * mq_copy_sql_hint  () - copy sql hint from src to dest
+ *
+ * return: void
+ * parser(in):
+ * dest_query(in):
+ * src_query(in):
+ *
+ */
+static void
+mq_copy_sql_hint (PARSER_CONTEXT * parser, PT_NODE * dest_query, PT_NODE * src_query)
+{
+  bool is_index_ss, is_index_ls;
+
+  if (dest_query->node_type == PT_SELECT)
+    {
+      is_index_ss = dest_query->info.query.q.select.hint & PT_HINT_INDEX_SS;
+      is_index_ls = dest_query->info.query.q.select.hint & PT_HINT_INDEX_LS;
+
+      /* merge HINT of vclass spec */
+      dest_query->info.query.q.select.hint =
+	(PT_HINT_ENUM) (dest_query->info.query.q.select.hint | src_query->info.query.q.select.hint);
+
+      dest_query->info.query.q.select.ordered =
+	parser_append_node (parser_copy_tree_list (parser, src_query->info.query.q.select.ordered),
+			    dest_query->info.query.q.select.ordered);
+
+      dest_query->info.query.q.select.use_nl =
+	parser_append_node (parser_copy_tree_list (parser, src_query->info.query.q.select.use_nl),
+			    dest_query->info.query.q.select.use_nl);
+
+      dest_query->info.query.q.select.use_idx =
+	parser_append_node (parser_copy_tree_list (parser, src_query->info.query.q.select.use_idx),
+			    dest_query->info.query.q.select.use_idx);
+
+      if (!is_index_ss || dest_query->info.query.q.select.index_ss != NULL)
```

Comment:
In the policy using hints, when INDEX_LS and INDEX_SS hints are specified at the same time, one hint is ignored.
INDEX_LS seems to be set only when (PT_HINT_NO_INDEX_SS or NOT PT_HINT_INDEX_SS).

Please refer to pt_print_select() and qo_get_ils_prefix_length().

Reply:
Okay. I'll fix it.